php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
How about we add a `php-cli.ini` with `display_errors=STDERR`, and then I'll be happy. This means nothing changes for web configuration, or for scripts ran with `exec`, but will output helpful messages for users running scripts on the CLI.